packages/plugin-apex/src/commands/force/apex/execute.ts

```diff
@@ -35,8 +35,6 @@ export default class Execute extends SfdxCommand {
     `$ sfdx force:apex:execute \nStart typing Apex code. Press the Enter key after each line, then press CTRL+D when finished.`
   ];
   protected static requiresUsername = true;
-  // Guaranteed by requires username
-  protected org!: Org;
```

Comment:
Note that upgrading `@salesforce/core` will results in a compile error if keeping this property:
> Property 'org' in type 'Execute' is not assignable to the same property in base type 'SfdxCommand'

Reply:
Does this have any impact on the usage of this property?

Reply:
I don't believe so.